File: Windows Phone 8.1/2.5.7.0/Decompilation/Vine.WindowsPhone/Vine.Models.Analytics/HttpRequestDetails.cs
using Newtonsoft.Json;

namespace Vine.Models.Analytics;

public class HttpRequestDetails
{
	[JsonProperty("method")]
	public string Method { get; set; }

	[JsonProperty("url")]
	public string Url { get; set; }

	[JsonProperty("http_status")]
	public long HttpStatus { get; set; }

	[JsonProperty("network_error")]
	public NetworkErrorCode? NetworkError { get; set; }

	[JsonProperty("os_error_details")]
	public string OsErrorDetails { get; set; }

	[JsonProperty("api_error")]
	public int ApiError { get; set; }
}

File: Android/Vine-5.50.1/Decompilation/src/java/co/vine/android/scribe/model/HTTPRequestDetails.java
package co.vine.android.scribe.model;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
/* loaded from: classes.dex */
public class HTTPRequestDetails {

    @JsonField(name = {"api_error"})
    public Integer apiError;

    @JsonField(name = {"http_status"})
    public Integer httpStatus;

    @JsonField(name = {"method"})
    public String method;

    @JsonField(name = {"network_error"})
    public String networkError;

    @JsonField(name = {"os_error_details"})
    public String osErrorDetails;

    @JsonField(name = {"url"})
    public String url;
}
